module/preference/PreferenceDialog.kt

```diff
@@ -346,16 +350,23 @@ object PreferenceDialog {
                 caption = PREFERENCES_GRAPH_OUTPUT_CAPTION
             )
 
-            override val preferences: List<PreferenceField> = listOf(graphOutput)
+            private val graphThemes = listOf(Color.Themes.DARK_GRAPH, Color.Themes.LIGHT_GRAPH)
```

Comment:
I feel like the Preferences dialog shouldn't be the source of truth for what graph themes are available in the application.

Suppose some intrepid community contributor submits their own colour theme, maybe a high contrast theme for the visually impaired. They will want to contribute their code to the themes package / file.

Therefore, we should make that file (`framework/common/theme/Theme.kt`) expose the list of all themes.

Having said that, that could potentially take away an important aspect of the list: the fact that they are displayed in the order given. I would recommend making this explicit - in the variable name, in an ideal world; but I'm not sure it will lend itself naturally to being named such, so perhaps a code comment is better.